eng/versioning/version_client.txt

```diff
@@ -0,0 +1,28 @@
+# Format;
+# groupId:artifactId;dependency-version;current-version
+
+com.azure:azure-sdk-parent;1.4.0;1.4.0
```

Comment:
Just out of curiosity if we wanted to add spacing to make this into a table for readability would that mess up your scripts?

Reply:
I split on the semicolon and could trim before assignment but honestly, it that even remotely worth it? With the varying lengths of the pieces (mainly artifacts and versions) I don't really see a lot of point in trying to format a plain text as a table.

Reply:
probably not, especially if we don't expect folks to hand author this file much. Even if we did enable this and people formatted it nicely we would then need to have your script that updates it round trip that formatting as well, which probably isn't worth the effort.